src/index.js

```diff
@@ -31,6 +32,14 @@ const retrieveVisitorToken = (config = {}) => {
   return token;
 };
 
+const retrievePageUUIDToken = () => {
+  let token = Cookie.get(PAGE_UUID);
```

Comment:
let's not fix this now cause I know we already did QA -- but I think this function can be one line
```
  return Cookie.get(PAGE_UUID) || uuid();
```